vector/src/main/java/im/vector/activity/VectorRoomCreationActivity.java

```diff
@@ -161,12 +161,8 @@ public void launchInviteMembersActivity(RoomCreationModes mode, VectorRoomInvite
         intent.putExtra(VectorRoomInviteMembersActivity.EXTRA_HIDDEN_PARTICIPANT_ITEMS, mParticipants);
         intent.putExtra(EXTRA_ROOM_CREATION_ACTIVITY_MODE, mode);
         intent.putExtra(VectorRoomInviteMembersActivity.EXTRA_INVITE_CONTACTS_FILTER, contactsFilter);
+        startActivity(intent);
```

Comment:
I would keep startActivityResult if mode.equals(RoomCreationModes.NEW_ROOM)

Reply:
It's useless here. In case of NEW_ROOM creation, we no longer use the VectorRoomCreationActivity but the TchapRoomCreationActivity.

Reply:
I know but this is to handle correctly NEW_ROOM mode in this activity even if this is not used